examples/basic-tracer/src/main.rs

```diff
@@ -0,0 +1,17 @@
+use opentelemetry::sdk::export::trace::stdout;
+use opentelemetry::sdk::trace::{self, Sampler};
+use opentelemetry::{trace::Tracer, KeyValue};
+
+fn init_tracer() -> impl Tracer {
+    let v = vec![KeyValue::new("key", "value")];
+    stdout::new_pipeline()
+        .with_trace_config(trace::config().with_sampler(Sampler::AlwaysOn))
+        .install_with_tracer_attributes(v)
```

Comment:
This would conflict with https://github.com/open-telemetry/opentelemetry-rust/pull/1027/files. 
Need to merge one first and tweak the other accordingly.

Reply:
I can remove the basic example for now.